lib/register_client.rb

```diff
@@ -0,0 +1,113 @@
+require 'rest-client'
+require 'json'
+
+module OpenRegister
+  class RegisterClient
+    def initialize register, phase
+      @register = register
+      @phase = phase
+
+      @items = []
+      @entries = { user: [], system: [] }
+      @records = { user: {}, system: {} }
+
+      refresh_data
+    end
+
+    def refresh_data
+      rsf = download_rsf(@register, @phase)
+      parse_rsf rsf
+    end
+
+    def download_rsf(register, phase)
+      RestClient.get('https://' + register + '.' + phase + '.openregister.org/download-rsf')
```

Comment:
should we use [string interpolation](https://stackoverflow.com/documentation/ruby/834/strings/2819/difference-between-single-quoted-and-double-quoted-string-literals#t=201709131624075680173) for this?

Reply:
Let's do it.